tests/remotes/gdrive.py

```diff
@@ -45,7 +108,7 @@ def gdrive(make_tmp_dir):
     # NOTE: temporary workaround
     tmp_dir = make_tmp_dir("gdrive", dvc=True)
 
-    ret = GDrive(GDrive.get_url())
-    tree = GDriveTree(tmp_dir.dvc, ret.config)
+    ret = GDrive(GDrive.get_url(), tmp_dir.dvc)
+    tree = ret.tree
```

Comment:
Shame gdrive is so complex that we have to reulse GdriveTree in our test fixtures :slightly_frowning_face: But oh well...